modules/appeals_api/spec/requests/v2/contestable_issues_controller_spec.rb

```diff
@@ -4,42 +4,24 @@
 require_relative AppealsApi::Engine.root.join('spec', 'support', 'shared_examples_contestable_issues.rb')
 
 describe AppealsApi::V2::DecisionReviews::ContestableIssuesController, type: :request do
-  include_examples 'contestable issues index requests',
-                   decision_review_type: 'higher_level_reviews',
-                   benefit_type: 'compensation',
-                   version: 'v2'
+  include_examples 'Contestable Issues API v0 and Decision Reviews v1 & v2 shared request examples',
+                   base_path: '/services/appeals/v2/decision_reviews/contestable_issues'
 
-  describe 'using versioned namespace route with oauth' do
-    include_examples 'contestable issues index requests',
-                     decision_review_type: 'higher_level_reviews',
-                     benefit_type: 'compensation',
-                     use_versioned_namespace_route: true,
-                     version: 'v2'
+  context 'with errors' do
+    let(:body) { JSON.parse(response.body) }
+    let(:headers) { { 'X-VA-SSN': 'abcdefghi', 'X-VA-Receipt-Date': '2019-12-01' } }
+    let(:path) do
+      '/services/appeals/v2/decision_reviews/contestable_issues/higher_level_reviews?benefit_type=compensation'
+    end
 
-    it_behaves_like(
-      'an endpoint with OpenID auth',
```

Comment:
I moved these openid auth specs into the shared examples as well, so that they will run automatically for any controller that defines `OAUTH_SCOPES`